src/PlutoTeachingTools.jl

```diff
@@ -2,10 +2,12 @@ module PlutoTeachingTools
 
 using PlutoUI
 using HypertextLiteral  # for aside, but might be useful elsewhere
+using Markdown
 
 using PlutoLinks: @ingredients
 export @ingredients
 
+include("i18n.jl")
```

Comment:
I'd like to add a comment like
# provides framework for multiple language support
for people who don't recognize i18n.

Reply:
Done!